accessibility-assessment.json

```diff
@@ -3,6 +3,10 @@
     {
       "path": "/test-only",
       "reason": "Test only endpoints do not need accessibility testing as it is used purely for testing purposes."
+    },
+    {
+      "path": "/non-existent-page",
+      "reason": "This path is used to check for a non-existent page, but many status errors would fail this check as they have the same elements"
     }
   ]
 }
```

Comment:
I don't think we need to do now, but I think test only routes are excluded by default now so might not need that, and then wonder if we could remove the need to have non existent page excluded too if we prefixed the url with test only too